### ISSUE TITLE
Go to function definition (Ctrl+Click) does not work in Javascript code (version 1.14)

### ISSUE THREAD

**Issue by v4umak** [Microsoft/vscode#30678]
- VSCode Version: 1.14
- OS Version: Windows 7

Steps to Reproduce:
In version 1.13 go to function definition (Ctrl+Click) worked. 
In new version - it does not. F12 does not work too. I've checked on simple file (named test.js):

```js
function testFunction(params) {
    alert("test");
}

testFunction();
```

To be sure I've removed old version and installed the new one - result was the same.

Reproduces without extensions: Yes

**Comment by mjbvz** [comment]
How did you open the file? Is it part of the workspace? If so, what does the rest of the workspace look like?

Can you also please open the output panel and check the `TypeScript` section for error messages.

**Comment by v4umak** [comment]
I've opened single js file and see no errors - [screenshot](http://joxi.ru/KAgeoopig4Jz4m)

**Comment by dreszczyk** [comment]
I have the same problem, tested with `.js` and `.jsx` files.

**Comment by mjbvz** [comment]
I believe this is a duplicate of #30716

Please run the `Toggle Developer tools` command and look in the developer console to confirm this. This bug should already be fixed in insiders and we are preparing a 1.14.2 update that fixes this as well

**Comment by v4umak** [comment]
Yes, I see the same error. Thank you!
